### ISSUE TITLE
FNX3-14040 ⁃ [Bug] X icon in collection should be "Remove from collection", not "Close tab"

### ISSUE THREAD

**Issue by NotWoods** [mozilla-mobile/fenix#10632]
## Steps to reproduce
* Navigate to collection using TalkBack
* Open collection
* Navigate to an "X" button inside one of the tabs of the collection

### Expected behavior
The content description that is read out should be something like "Remove from collection", implying that the button removes the tab from the collection.

### Actual behavior
The content description is "Close tab", implying that it is an already open tab that can be closed.

Once we have UX approval for the strings change, this can be turned into a good first issue.

**Comment by ekager** [comment]
For content descriptions, eng can usually just make a new string that makes is succinct and makes sense especially when the current description is just plain wrong.

**Comment by partha2000** [comment]
Can I work on this issue? As it is a "good first issue" it would be motivating for me to get started in the contributing process.

**Comment by BMEduard** [comment]
Sure @partha2000 , that would be great, you can go ahead and start working on this issue.

**Comment by partha2000** [comment]
@BMEduard I have submitted a pull request #10663 to solve this issue. Could you review the PR it if it is ok to be merged?

**Comment by partha2000** [comment]
As of now the PR #10673 has solved the issue. But this issue is still open as the `contentDescription` can still be modified to readout the name of the site.

**Comment by brampitoyo** [comment]
@partha2000 Thanks for fixing this issue. “Remove tab from collection” is definitely better than “Remove from collection”.

**Comment by sflorean** [comment]
Verified as fixed on Nightly 6/15 with Samsung Galaxy Note 10 (Android 10) and Nokia 6 (Android 7.1.1).
